server/src/main/kotlin/suwayomi/tachidesk/manga/impl/update/Updater.kt

```diff
@@ -29,6 +31,9 @@ class Updater : IUpdater {
     private val tracker = ConcurrentHashMap<Int, UpdateJob>()
     private val updateChannels = ConcurrentHashMap<String, Channel<UpdateJob>>()
 
+    // max parallel update requests
+    private val semaphore = Semaphore(10)
```

Comment:
Is it viable for this constant to be available for the config file?

Reply:
done